FiveCalls/FiveCalls/UserLocation.swift

```diff
@@ -50,10 +50,10 @@ class UserLocation {
         }
     }
     
-    func setFrom(zipCode: String) {
-        locationType = .zipCode
-        locationValue = zipCode
-        locationDisplay = zipCode
+    func setFrom(address: String) {
+        locationType = .address
+        locationValue = address
+        locationDisplay = address
```

Comment:
Here we have to consider the case where the user had `zipCode` in user defaults already, then runs the app w/ this code later on. I think we should probably have a `Migrations` class that handles this on startup, that can manually check, something like this:

```swift
class Migrations {
  func run() {
    migrateUserDefaultsFromZipCodeToAddress()
  }

  private func migrateUserDefaultsFromZipCodeToAddress() {
     ...
  }
}
```

We can call this `run()` function early on in the `applicationDidFinishLaunching` method.